modules/encrypt_seclib/src/Plugin/EncryptionMethod/PHPSecLibEncryption.php

```diff
@@ -13,7 +13,8 @@
  * @EncryptionMethod(
  *   id = "phpseclib",
  *   title = @Translation("PHP Secure Communications Library (phpseclib)"),
- *   description = "Uses the <a href='http://phpseclib.sourceforge.net/'>phpseclib</a> library. This method is only preferable if you cannot install mcrypt."
+ *   description = "Uses the <a href='http://phpseclib.sourceforge.net/'>phpseclib</a> library. This method is only preferable if you cannot install mcrypt.",
+ *   key_type = {"aes_encryption"}
```

Comment:
Is this right? Thought we were settling on "encryption" in lieu of "aes_encryption"